java/code/src/com/suse/manager/webui/websocket/WebsocketSessionConfigurator.java

```diff
@@ -35,12 +31,6 @@ public void modifyHandshake(ServerEndpointConfig config,
             HandshakeRequest request,
             HandshakeResponse response) {
         HttpSession httpSession = (HttpSession)request.getHttpSession();
-        HttpServletRequest sr = (HttpServletRequest) httpSession.getAttribute("__original_request__");
-        User user = new RequestContext(sr).getCurrentUser();
-        // force roles to be loaded so now since it will fail from the websocket thread
-        if (user != null) {
-            user.getRoles();
-            config.getUserProperties().put("currentUser", user);
-        }
+        config.getUserProperties().put("webUserID", httpSession.getAttribute("webUserID"));
```

Comment:
Similar to the above: do we have guarantees over `getUserProperties` not returning `null`?

Reply:
The documentation doesn't say it can return null, so I'm not expecting it could. But in the above, the documentation says explicitly that it never returns null.
Documentation: "This method returns a modifiable Map that the developer may use ..." (https://docs.oracle.com/javaee/7/api/javax/websocket/EndpointConfig.html#getUserProperties--)
I think we don't need to protect against NPE in this one.